waku/waku.go

```diff
@@ -971,7 +968,7 @@ func (w *Waku) GetFilter(id string) *common.Filter {
 func (w *Waku) Unsubscribe(id string) error {
 	ok := w.filters.Uninstall(id)
 	if !ok {
-		return fmt.Errorf("Unsubscribe: Invalid ID")
+		return fmt.Errorf("failed to unsubscribe: invalid ID '%s'", id)
```

Comment:
error message broke convention for Go errors. Don't start with capital letters. Also made the error more meaningful by adding the id of the failed `Unsubcribe()`